privval/signer_client.go

```diff
@@ -68,7 +69,7 @@ func (sc *SignerClient) Ping() error {
 // GetPubKey retrieves a public key from a remote signer
 // returns an error if client is not able to provide the key
 func (sc *SignerClient) GetPubKey() (crypto.PubKey, error) {
-	response, err := sc.endpoint.SendRequest(mustWrapMsg(&privvalproto.PubKeyRequest{}))
+	response, err := sc.endpoint.SendRequest(mustWrapMsg(&privvalproto.PubKeyRequest{ChainId: sc.chainID}))
```

Comment:
can we add some flag for `ChainId` field to be `ChainID` instead? like `custom_name` or smth

Reply:
I can add it. The reason I went with avoiding it is if we decide to move away from gogo protobuf then all this field will break, this way it's avoiding the breakage.

Reply:
Ah, then lets leave it as is. Thanks